packages/compass-e2e-tests/package.json

```diff
@@ -9,14 +9,15 @@
     "lint": "npm run eslint . && npm run prettier -- --check .",
     "depcheck": "depcheck",
     "check": "npm run lint && npm run depcheck",
-    "pretest": "mongodb-runner start --port=27018 --dbpath ./.mongodb",
+    "pretest": "mongodb-runner start --port=27018 --dbpath ./.mongodb && npm run insert-data",
```

Comment:
Ended up just doing the simplest thing and making this a script. We already depend on mongodb elsewhere anyway. If we outgrow it we can replace it and go for one of the many options we discussed in slack.